docs/DeveloperGuide.md

```diff
@@ -487,23 +487,73 @@ Extensions:
-* **Portion**: A breakdown of a transaction into individual portions, each of which involves a single contact and a weightage indicating the proportion of the transaction cost that the contact needs to pay to the person who paid for the transaction.
-* **Balance**: The amount of money either owed to you or owed by you to another person in your contacts.
-  * **Positive Balance**: Indicates that the contact owes you money.
-  * **Negative Balance**: Indicates that you owe the contact money.
-* **Outstanding Balance**: The amount of unsettled money between you and your contact.
-* **Settle**: The action of clearing any outstanding balance between you and another contact via a new portion.
-* **Payee**: The person that paid the bill for that specific transaction
-
-**General**
-* **Mainstream OS**: Windows, Linux, Unix, OS-X.
-* **Command**: A text input from the user which tells the application to run a specific action.
+#### Address
+The particulars of the place where a person lives.
+#### Alphanumeric
+Consisting of only letters (a-z, A-Z) or numbers or both.
+#### Amount
+The total value of the transaction.
+#### Balance
+The amount of money that you owe a person or the amount of money that they owe you. A positive balance under a person
+means that they owe you money, whereas a negative balance means that you owe them money.
+#### Card
+A rectangular area in our application that either describes the full details of a transaction or person.
+#### Command
+To use and control the application, commands are necessary. To utilise a feature, a command has to be typed into the Command Input Field and executed.
+#### Cost
+The total value of the transaction.
+#### Description
+A written account of a transaction, used to provide context and details about the transaction.
+#### Email address
+A unique identifier for an email account. It identifies an email box to which messages and emails are delivered.
+#### Field
+An area where text can be input.
+#### Flag
+Flags are used to modify the operation of a command.
+#### Gui
+Gui stands for graphical user interface. A graphical user interface uses icons and mouse inputs from users to allow them to interact with the application.
+#### Index
+A number representing the position of an item in a list.
+#### Integer
+A number that is a not a fraction. E.g. whole numbers such as -10, 0, 1, 5.
+#### Keyword
+An important word that is used by the application to process various commands.
+#### Mainstream OS
+Refers to operating systems such as Windows, Linux, Unix, OS-X.
+#### Name
+A word or set of words that are used to address or refer to a person.
+#### Negative
+A numerical value that is less than 0.
+#### Parameter
+An additional input that provides further details on a command that a user is executing.
+#### Payee
+Refers to the person that paid for the transaction.
+#### Payer
+Refers to a person that owes the payee money for the transaction.
+#### Person
+A human being.
```

Comment:
How about instead we define it as something like:

refers to an individual that can be associated with various data within the app, such as contact information and expenses.

Reply:
updated to An individual that can be associated with various information in our application, such as contact information and balances.